docs/protocol/extension_generate.md

```diff
@@ -140,7 +140,7 @@ Content-Length: <xx>
 ### Generate Response JSON Object
 
 A successful generate request is indicated by a 200 HTTP status code.
-The generate response object, identified as *$generate_response*, is returned in
+The generate response object, identified as `$generate_response`, is returned in
```

Comment:
I don't think this change is necessary, since original version was good: 
![image](https://github.com/triton-inference-server/server/assets/124622579/ce2c7b17-0a73-4e6e-a204-5a4d80f513bb)

Reply:
Actually, no worries, let's keep this for consistency